addOns/sstiscanner/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this add-on will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## First release
+
+- Add a scanner to find reflected/stored SSTI and one to find  blind SSTI. 
```

Comment:
I somehow wasn’t clear in my comments on this.

```md
## Unreleased

- Add a scanner to find reflected/stored SSTI and one to find  blind SSTI. 
```

The ‘unreleased’ header will be changed as part of the release gradle tasks.

Reply:
No problem. Do you need me to do something else?

Reply:
That's the only thing I see.

I haven't been able to grab this and test it yet but I'm sure you've done lots on that front.

Reply:
I tested it several times. I have some tests on https://github.com/DiogoMRSilva/websitesVulnerableToSSTI if you need some vulnerable examples to test it.